client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -24,18 +24,19 @@ const LabelWrapper: FunctionComponent< LabelWrapperProps > = ( { label, children
 	);
 };
 
-interface Props {
+interface GSuiteNewUserProps {
 	autoFocus: boolean;
 	domains: string[];
 	onUserRemove: () => void;
 	onUserValueChange: ( field: string, value: string, mailBoxFieldTouched?: boolean ) => void;
 	onReturnKeyPress: ( event: Event ) => void;
 	selectedDomainName: string;
 	showTrashButton: boolean;
+	showAllErrors?: boolean;
 	user: NewUser;
 }
 
-const GSuiteNewUser: FunctionComponent< Props > = ( {
+const GSuiteNewUser: FunctionComponent< GSuiteNewUserProps > = ( {
```

Comment:
Minor comment: as above, we could remove `FunctionComponent` here.
```suggestion
const GSuiteNewUser = ( {
  [props...]
}: GSuiteNewUserProps ): ReactElement => {
```